internal/controllers/synthesis/exec.go

```diff
@@ -137,8 +135,7 @@ func (c *execController) fetchPreviousSlices(ctx context.Context, comp *apiv1.Co
 		slice.Name = ref.Name
 		slice.Namespace = comp.Namespace
 		err := c.noCacheClient.Get(ctx, client.ObjectKeyFromObject(slice), slice)
-		if errors.IsNotFound(err) && comp.Status.PreviousSynthesis.Synthesized != nil && time.Since(comp.Status.PreviousSynthesis.Synthesized.Time) > time.Minute*5 {
```

Comment:
This isn't possible now since we've fixed the manager - we're making real calls to apiserver instead of hitting the informer cache.